src/lib/meshCmds/main.go

```diff
@@ -15,7 +16,7 @@ const GetOnlineStatusListCmd uint16 = 2100
 const createRemoteRoomCmd uint16 = 2000
 
 func Setup() {
-	mesh.Command(createRemoteRoomCmd, handleCreateRemoteRoom)
+	mesh.HandleCommand(createRemoteRoomCmd, handleCreateRemoteRoom)
```

Comment:
SHOULD: ユーザーがカスタムで使うのは `diarkisexec.SetMeshCommandHandler()` のほうに置き換えていきたいです
https://docs.diarkis.io/docs/server/current/diarkis/diarkisexec/index.html#hdr-Declaring_Custom_Mesh_Command_Handlers__Server_to_Server_Command_Handlers_

```suggestion
	diarkisexec.SetMeshCommandHandler(createRemoteRoomCmd, handleCreateRemoteRoom)
```